bindings/python/src/session.cpp

```diff
@@ -683,9 +715,9 @@ void bind_session()
         .def("set_dht_settings", allow_threads(&lt::session::set_dht_settings))
         .def("get_dht_settings", allow_threads(&lt::session::get_dht_settings))
         .def("dht_get_immutable_item", allow_threads(dht_get_immutable_item))
-        .def("dht_get_mutable_item", allow_threads(dht_get_mutable_item))
+        .def("dht_get_mutable_item", &dht_get_mutable_item)
```

Comment:
libtorrent/bindings/python/src/gil.hpp:69:30: error: return-statement with a value, in function returning 'void' [-fpermissive]
         return (a0.*fn)(a1,a2);

allow_threads does not work for functions that dont return anyting